ToolTip.ios.js

```diff
@@ -24,15 +24,19 @@ var propTypes = {
 var ViewClass = React.createClass({
   getDefaultProps: function() {
     return {
-      arrowDirection: 'down'
+      arrowDirection: 'down',
+      onHide: () => true,
+      onShow: () => true
     };
   },
 
   showMenu: function() {
     ToolTipMenu.show(findNodeHandle(this.refs.toolTipText), this.getOptionTexts(), this.props.arrowDirection);
+    this.props.onShow();
```

Comment:
This is true for programmatically displaying the tooltip but there's no way to attach an event when you use show it using the longPress. I think there should be e.g. if you want to change the style when the tooltip is displayed.